eng/tools/GenerateFiles/Directory.Build.props.in

```diff
@@ -2,5 +2,6 @@
   <PropertyGroup>
     <DefaultNetCoreTargetFramework>${DefaultNetCoreTargetFramework}</DefaultNetCoreTargetFramework>
     <TreatWarningsAsErrors Condition="'$(BuildingInsideVisualStudio)' != 'true'">true</TreatWarningsAsErrors>
+    <AppHostPackVersion Condition=" '$(AppHostPackVersion)' == '' ">${AppHostPackVersion}</AppHostPackVersion>
```

Comment:
Please remove this and the change in GenerateFiles.csproj. See #39965

Reply:
Ah, I'm starting to get this based on your comments in src/Servers/IIS/Directory.Build.props. Suspect I made the suggestion and got confused by the property name. Let's at least change the `$(AppHostPackVersion)` property name so that it doesn't sound like something we're overriding in the .NET SDK. Make it visibly unique to avoid similar confusion in the future.

A comment like "for `$(LibNetHostPath)` setting in src/Servers/IIS/Diretory.Build.props" would also help.

Reply:
Right this was your suggestion to get things to flow for native projects, so what do you want this name to be?

Reply:
Could go with LibNetHostAppPackVersion?

Reply:
> Could go with LibNetHostAppPackVersion?

Works for me